umbraco/Umbraco-CMS
# Fixes v8: Correct label filter is applied when copying NC

Fixes: #6162 

The reason for the issue was that once Forms are installed `truncate` filter on Forms was overwriting the one on the CMS which required different parameters. So, after talking with @nielslyngsoe, we agreed on changing the name of the filter on the CMS, and thus resolving the problem.

Testing:
- Create a nested content data type or use an existing one in the following step
- Add it to a document type
- Create some content
- Copy the content
- Click "Add Content" again and see how the content is named

Result:
- You should be able to see the full name of the copied item, or part of it and then "..." if it is too long
